### ISSUE TITLE
about cfm training details

### ISSUE THREAD

**Issue by amyburden** [SWivid/F5-TTS#760]
### Checks

- [x] This template is only for question, not feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I confirm that I am using English to submit this report in order to facilitate communication.

### Question details

Hi, Thanks so much for your work. It looks very promising. But I have a little confusion about the flow matching details.
in file ```f5_tts/model/cfm.py``` line 256. 
```
φ = (1 - t) * x0 + t * x1
flow = x1 - x0
```
and final loss is ```loss = F.mse_loss(pred, flow, reduction="none")```.
My question is that transformer intends to model noise between φ(input) and x1. Why does the flow= x1-x0 instead of x1-φ.
